aerospike-core/Cargo.toml

```diff
@@ -8,13 +8,13 @@ edition = "2018"
 [dependencies]
 log = "0.4"
 byteorder = "1.3"
-ripemd160 = "0.8"
-base64 = "0.11"
-crossbeam-queue = "0.2"
-rand = "0.7"
+ripemd = "*"
```

Comment:
This is a more actively maintained package.... the other one gave us audit errors.